docs/guides/host.md

```diff
@@ -30,13 +30,13 @@ prefect server start
 
 Open the URL for the Prefect server UI ([http://127.0.0.1:4200](http://127.0.0.1:4200) by default) in a browser.
 
-![Viewing the dashboard in the Prefect UI.](/img/ui/self-hosted-server-dashboard.png)
+![Viewing the dashboard in the Prefect UI.](/img/ui/self-managed-server-dashboard.png)
```

Comment:
we would need to change the name of the actual file to avoid breaking the link right?